shared/epaper/epaper.pro

```diff
@@ -1,32 +1,18 @@
 !linux-oe-g++{
     error("epaper is only supported on the reMarkable")
 }
-TEMPLATE = lib
-QT += core
-QT += quick
+TEMPLATE = aux
 
-include(../../qmake/common.pri)
-
-VERSION = 1.0
-
-DISTFILES = \
-    epframebuffer.h
-
-TARGET = qsgepaper
-target.path = /opt/lib
-INSTALLS += target
-
-
-epframebuffer_h.target = epframebuffer.h
-epframebuffer_h.commands = cp $$PWD/epframebuffer.h $$OUT_PWD
+epframebuffer_h.target = raw_copy
+epframebuffer_h.commands = cp $$PWD/epframebuffer.h $$PWD/libqsgepaper.so $$OUT_PWD
 QMAKE_EXTRA_TARGETS += epframebuffer_h
 
 linux-oe-g++{
-    epframebuffer_h_install.files = epframebuffer.h
+    epframebuffer_h_install.files = epframebuffer.h libqsgepaper.so
```

Comment:
Libqsgepaper should be it's own install that places it in the correct location, right now this is placing it in /opt/include instead of /opt/lib/plugins/scenegraph